### PR TITLE
release fixes 1.8.1

### LINKED ISSUE #967: incorrect first time paying
flow:
- send
- recieve from them
- go to send again
- “first time paying X” incorrectly

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
- make AASA bug error more helpful

- fix last send / receive population in contact, closes #967 

tested by running a couple phones side by side and trying a few combinations

image

### CODE REVIEW

File: apps/daimo-mobile/src/logic/daimoContacts.ts [hunk 1]
```diff
@@ -167,16 +161,13 @@ export function useContactSearch(
...
-          lastSendTime: recent.lastSendTime,
-        });
+        recipients.push({ ...recent, originalMatch: account.originalMatch });
       } else {
         recipients.push({ type: "eAcc", ...account });
       }
     }
-    const sortKey = (r: DaimoContact) => r.lastSendTime || 0;
+    const sortKey = (r: DaimoContact) =>
+      Math.max(r.lastSendTime || 0, r.lastRecvTime || 0);
```
Comment (dcposch) [inline apps/daimo-mobile/src/logic/daimoContacts.ts:170]: 🚀

File: apps/daimo-mobile/src/logic/passkey.ts [hunk 2]
```diff
@@ -15,12 +15,14 @@ import { env } from "./env";
...
 import { parseCreateResponse, parseSignResponse } from "./passkeyParsers";
 
-// Workaround iOS deeplinks bug: https://github.com/daimo-eth/daimo/issues/837
+// Workaround iOS Passkeys AASA bug: https://github.com/daimo-eth/daimo/issues/837
 function matchAASABugError(e: string) {
   // Match without english text since iOS errors are localized to device language
   return e.includes("JV8PYC9QV4.com.daimo") && e.includes("daimo.com");
 }
 
+const AASA_BUG_MESSAGE = "iOS system error, restart the app and try again.";
```
Comment (dcposch) [inline apps/daimo-mobile/src/logic/passkey.ts:24]: Maybe 
> iOS system error. This is a known issue right after install. Restart the app, then try again.

**Review by dcposch — APPROVED** [review]
LGTM

File: apps/daimo-mobile/src/logic/passkey.ts (hunk 2 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/logic/passkey.ts]: done

**Pull request merged**